include/elf.h

```diff
@@ -1506,12 +1506,13 @@ typedef struct elf64_sym {
 } Elf64_Sym;
 
 typedef struct elf128_sym {
+  Elf128_Addr st_value;		/* Value of the symbol */
```

Comment:
I would rather put st_value and st_size after st_name (1st field of both elf32 and elf64 sym struct) and after st_info and st_other (kind of like elf64_sym). This is clearly cosmetic, but keeps at least the 1st field of the struct identical for all sizes.